### ISSUE TITLE
win32ax.inc not found

### ISSUE THREAD

**Issue by WeaponizedMeme** [x0r19x91/010-Editor-k3yg3n#1]
Hi, I downloaded Flat Assembler and compiled with the instructions you wrote. I however got this error and am unsure why. I remember you used to have  compiled binaries available, would be really helpful! :)
![image](https://user-images.githubusercontent.com/37276321/48674536-f65c8f80-eb12-11e8-9e1f-510423ed0869.png)

**Comment by x0r19x91** [comment]
Yes I'll release binaries shortly... Sorry for the inconvenience

**Comment by x0r19x91** [comment]
I've released the binaries
